pipelines/build/common/trigger_beta_build.groovy

```diff
@@ -114,6 +114,47 @@ def loadTargetConfigurations(String javaVersion, String variant, String configSe
     return targetConfigurationsForVariant
 }
 
+// Verify the given published release tag contains the given asset architecture
+def checkJDKAssetExistsForArch(String binariesRepo, String version, String releaseTag, String variant, String arch) {
```

Comment:
```suggestion
def checkJDKAssetExistsForArch(String binariesRepo, String version, String releaseTag, String arch) {
```
To make linter happy, as "variant" is unused.

Reply:
yep removed